terasoluna-gfw-functionaltest-web/src/main/java/org/terasoluna/gfw/functionaltest/app/codepoints/ContainsAllForm1.java

```diff
@@ -1,3 +1,18 @@
+/*
+ * Copyright (C) 2013-2016 terasoluna.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */

```

Comment:
@btsatoukzn 
Are these different from the old file copyrights?
https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/blob/master/terasoluna-gfw-functionaltest-web/src/main/java/org/terasoluna/gfw/functionaltest/app/codepoints/ConsistOfCheckForm.java#L2
```java
/*
 * Copyright (C) 2013-2016 NTT DATA Corporation
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
 * either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
 */
```